source/validate_instruction.cpp

```diff
@@ -205,6 +205,12 @@ spv_result_t ReservedCheck(ValidationState_t& _,
       return _.diag(SPV_ERROR_INVALID_BINARY)
              << "Invalid Opcode name 'Op" << inst_desc->name << "'";
     }
+    case SpvOpUndef:
```

Comment:
This function is called CheckReserved, should the name be updated to reflect that we also do env-related checks?

Reply:
Also the comment on the method would need updating.

The alternative is to make a whole new function.

Reply:
Chose to add a new EnvironmentCheck function.